torch/testing/_internal/common_methods_invocations.py

```diff
@@ -280,6 +311,60 @@ def __init__(self,
         #   outside a function's domain.
         self._domain_eps = 1e-5
 
+    def sample_sparse_inputs(self, device, dtype, requires_grad=False, **kwargs):
+        """Returns an iterable of SampleInputs."""
+        if not self.supports_sparse:
+            raise RuntimeError("This op does not support sparse input")
+
+        low, high = self.domain
+        low = -10 if low is None else low + self._domain_eps
+        high = 10 if high is None else high - self._domain_eps
+        is_coalesced = kwargs.get("is_coalesced", False)
+
+        numel = 10
+        t1 = torch.sparse_coo_tensor(
+            indices=torch.arange(numel).unsqueeze(0),
```

Comment:
1D is interesting, but 2D with a density target of ~20% is probably more "representative"